integration_test/third_party_apps_data/applications/cassandra/debian_ubuntu/install

```diff
@@ -1,7 +1,20 @@
 set -e
 
 # Required to install cassandra
-echo "deb http://www.apache.org/dist/cassandra/debian 22x main" | sudo tee -a /etc/apt/sources.list.d/cassandra.sources.list
+source /etc/os-release
+if [[ "${VERSION_CODENAME}" = "lunar" ]]; then
+    # There is a python dependency that fails for any older versions of cassandra on ubuntu 23
```

Comment:
Are we installing Cassandra 4.1x here? Might want to explicitly mention that in the comment.